### PR TITLE
Improve bare metal compute node management

### PR DESCRIPTION
**Pull request by markgoddard**
Adds compute-node-manage.yml and compute-node-inspect.yml

### CODE REVIEW

File: ansible/baremetal-compute-inspect.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- name: Ensure baremetal compute nodes are inspected in ironic
+  hosts: baremetal-compute
+  gather_facts: False
+  vars:
+    controller_host: "{{ groups['controllers'][0] }}"
+    # Whether to wait for the state transition to complete.
+    baremetal_compute_wait: True
+    # Time to wait for state transition to complete, if baremetal_compute_wait
+    # is True.
+    baremetal_compute_timeout: 1200
```
Comment (dougszumski) [inline ansible/baremetal-compute-inspect.yml:30]: The cleaning steps shouldn't run during inspection, right?

File: doc/source/administration.rst [hunk 2]
```diff
@@ -125,6 +125,32 @@ any of these hosts are not expected to be active (e.g. prior to overcloud
...
+Baremetal Compute Node Management
+=================================
+
+When enrolling new hardware or performing maintenance, it can be useful to be
+able to manage many bare metal compute nodes simulteneously.
+
+In all cases, commands are delegated to one of the controller hosts, and
+executed concurrently. Note that ansible's ``forks`` configuration option,
+which defaults to 5, may limit the number of nodes configured concurrently.
+
```
Comment (dougszumski) [inline doc/source/administration.rst:137]: Should we mention the limitations here, e.g., it doesn't support moving an active node to the manageable state?

File: ansible/baremetal-compute-inspect.yml (hunk 1 above)
Comment (markgoddard) [reply ansible/baremetal-compute-inspect.yml:30]: No, they'll only run during cleaning.

File: doc/source/administration.rst (hunk 2 above)
Comment (markgoddard) [reply doc/source/administration.rst:137]: Added some words.

**Review by dougszumski — APPROVED** [review]
Looks good!

**Pull request merged**